v2/internal/platform/win32/structs.go

```diff
@@ -0,0 +1,54 @@
+package win32
+
+type NOTIFYICONDATA struct {
+	CbSize           uint32
+	HWnd             HWND
+	UID              uint32
+	UFlags           uint32
+	UCallbackMessage uint32
+	HIcon            HICON
+	SzTip            [128]uint16
+	DwState          uint32
+	DwStateMask      uint32
+	SzInfo           [256]uint16
+	UVersion         uint32
+	SzInfoTitle      [64]uint16
+	DwInfoFlags      uint32
+	GuidItem         GUID
+	HBalloonIcon     HICON
+}
+
+type GUID struct {
```

Comment:
IIRC this is a duplicate definition in another package, maybe also using `windows.GUID`?